ext/fetch/20_headers.js

```diff
@@ -59,17 +58,7 @@
    * @returns {string}
    */
   function normalizeHeaderValue(potentialValue) {
-    potentialValue = StringPrototypeReplaceAll(
-      potentialValue,
-      HTTP_WHITESPACE_PREFIX_RE,
-      "",
-    );
-    potentialValue = StringPrototypeReplaceAll(
-      potentialValue,
-      HTTP_WHITESPACE_SUFFIX_RE,
-      "",
-    );
-    return potentialValue;
+    return httpTrim(potentialValue);
```

Comment:
Hi! Sorry about my ignorance, but why not just move the entire content of `httpTrim` to `normalizeHeaderValue` function? That'd remove `httpTrim` definition everywhere which is a lot of lines less with the same result